src/blueprints/tv2/factories/tv2-transition-factory.ts

```diff
@@ -0,0 +1,110 @@
+import { Action, MutateActionMethods, PieceAction } from '../../../model/entities/action'
+import { ActionType } from '../../../model/enums/action-type'
+import { Piece, PieceInterface } from '../../../model/entities/piece'
+import { PieceType } from '../../../model/enums/piece-type'
+import { TransitionType } from '../../../model/enums/transition-type'
+import { PieceLifespan } from '../../../model/enums/piece-lifespan'
+import { Tv2AtemLayer, Tv2SourceLayer } from '../value-objects/tv2-layers'
+import { AtemMeTimelineObject, AtemTransition, AtemType } from '../../timeline-state-resolver-types/atem-types'
+import { DeviceType } from '../../../model/enums/device-type'
+import { Tv2BlueprintTimelineObject } from '../value-objects/tv2-meta-data'
+import { TimelineObject } from '../../../model/entities/timeline-object'
+
+const FRAME_RATE: number = 25
+
+const FRAMES_FOR_MIX_TRANSITION: number = 25 // TODO: Get from configuration.
+
+const MIX_TRANSITION_ID: string = 'nextTakeHasMixTransitionAction'
```

Comment:
Due to how this is used in `isTransitionAction(...)`, i get the sense that there might well exist other Mix transition ids in the future.

You might say YAGNI, but would it hurt to have this as an enum already?
The method i mentioned should then be able to easily be updated to get an array of elements in the enum, when the need arise in the future, or already now, if you want to make the change already.

If there isn't the possibility for more Mix transition ids, then why is the value put into an array in the method?

Reply:
Currently, there aren't any techincal limitations for using enum here. 
However, I am bit hesistant to use Enums to store ids.

> The method i mentioned should then be able to easily be updated to get an array of elements in the enum

The method already is easily updatable ;) 

> If there isn't the possibility for more Mix transition ids, then why is the value put into an array in the method?

You could argue YAGNI here, but there are going to be more transition ids. We still need to implement `dip` and `wipe` transitions. And we also might need to have a id for each transtion that is for next Part and one for active Part.

Reply:
What i am trying to get at, is that if the Id was stored in an enum, and the moethod was taking the values of the enum to check for, then the method would already handle adding new id, without having to make any changes to it.

I am fine with keeping it as it. Just my mind wanting to prepare for the possibility of more ids being added, ahead of time.